forms-shared/src/schemas/sekcia-uzemneho-rozvoja/surSchemas.ts

```diff
@@ -48,7 +48,7 @@ const ziadatelInvestorFields = [
   sharedPhoneNumberField('telefon', true),
 ]
 
-export const getSchema = (zavazne: boolean) =>
+export const getSurSchema = (zavazne: boolean) =>
```

Comment:
This seems to me as a thing which may become problematic one day. The schemas are not really the same, there are multiple ifs checking for this boolean and creating the schema according to it. However if these two forms are the only ones for which we use this approach, it is okay. Otherwise for clarity I would separate the two schemas and not use this function.

But I see this has already been implemented, so maybe just a thought for the future, if we want to make these schemas like this.